pyproject.toml

```
@@ -14,18 +14,18 @@ classifiers = [
 dependencies = [
     'asdf >=2.15.0',
     'asdf-astropy >=0.4.0',
-    'astropy >=5.0.4',
+    'astropy >=5.3.0',
     'crds >=11.16.16',
     'gwcs >=0.18.1',
     'jsonschema >=4.8',
     'numpy >=1.22',
     'photutils >=1.6.0',
     'pyparsing >=2.4.7',
     'requests >=2.22',
-    'rad == 0.17.0',
+    'rad == 0.17.1',
 #    'rad >= 0.16.0, < 0.17.1',
 #    'rad @ git+https://github.com/spacetelescope/rad.git@main',
-    'roman_datamodels ==0.17.0',
+    'roman_datamodels ==0.17.1',
```

```suggestion
    'roman_datamodels >=0.17.1',
```
This should not be exact pinned.

---

Why, what purpose does that serve for a release?

---

Exact pinning like this should be on a release branch not a development branch, if it needs to be done at all. Exact pins like these will interfere with development of `rad` and `roman_datamodels` by requiring us to make the changes I requested here in order to move forward with them. This is not a concern on a release branch, only on the development branch.